setup.py

```diff
@@ -40,8 +39,14 @@
     "hf_doc_builder @ git+https://github.com/huggingface/doc-builder.git",
 ]
 
+PYTEST_REQUIRES = [
+    "pytest",
+    "pytest-mock",
```

Comment:
@quic-jhugo and @carlstreeter-quic, Could you please review this addition of new package for test automation?

Reply:
As these are declared as extras, they are not installed with a typical installation.  Is that what's intended? Basically, they don't actually add anything to the default installation.

Reply:
@carlstreeter-quic , the idea was to enable installation of pytests* for test environment only, that's why we had to keep it in extras, in Jenkins we trigger installation via `pip install -e[test]`. Also anyone interested in running tests should do the same.